test/ttmlir/Dialect/TTNN/embedding/embedding_1d_tensor.mlir

```diff
@@ -2,7 +2,6 @@
 #any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
 module attributes {} {
   func.func @forward(%arg0: tensor<32xf32>, %arg1: tensor<512x128xf32>) -> tensor<32x128xf32> {
-    // CHECK: %[[C:.*]] = "ttnn.empty"[[C:.*]]
     %0 = tensor.empty() : tensor<32x128xf32>
     // CHECK: %[[C:.*]] = "ttnn.embedding"[[C:.*]]
```

Comment:
If you mean something like this?
`// CHECK: %0 =  "ttnn.embedding" [[C:.*]]`

I don't think we can do that since there can be meny ops preceding ttnn.embedding

Reply:
In this specific example, I meant to verify that there is no empty op in the ttnn IR...

Reply:
Short google shows `CHECK-NOT` directive as an option, could we try that?

Reply:
we can